packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/RetroTopic.tsx

```diff
@@ -133,23 +133,23 @@ const RetroTopic = (props: Props) => {
       {(topicSummary || discussionSummary) && (
         <tr>
           <td align='left' style={{lineHeight: '22px', fontSize: 14}}>
-            {topicSummary && (
+            {team?.tier === 'personal' && (
               <tr>
-                <td style={topicTitleStyle}>{'Topic Summary:'}</td>
+                <td
+                  style={explainationStyle}
```

Comment:
I've moved this around as I think the explanation sentence should come before the topic & discussion summary titles